tests/core/distortion/test_distortion_dexela_2923.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import pytest
+
+from hexrd.core.distortion.dexela_2923 import Dexela_2923, _find_quadrant
+import hexrd.core.distortion.dexela_2923 as dexela_2923
+
+
+@pytest.fixture(autouse=True)
+def bypass_numba_for_coverage(monkeypatch, request):
+    """When coverage is active, replace njit wrappers with their Python bodies.
+
+    This keeps the @njit decorators in the source but ensures pytest-cov
+    / Codecov see the Python lines by executing the original functions via
+    .py_func during coverage runs.
+    """
+    cov_plugin = request.config.pluginmanager.getplugin("pytest_cov")
+    cov_active = cov_plugin is not None or getattr(request.config, "cov_controller", None)
+
+    if cov_active:
+        monkeypatch.setattr(
+            dexela_2923,
+            "_dexela_2923_distortion",
+            dexela_2923._dexela_2923_distortion.py_func,
+        )
+        monkeypatch.setattr(
+            dexela_2923,
+            "_dexela_2923_inverse_distortion",
+            dexela_2923._dexela_2923_inverse_distortion.py_func,
+        )
+
+
+def test_dexela_2923_distortion_class():
+    params = [0.0] * 8
+    distortion_instance = Dexela_2923(params)
+
+    # Test properties
+    assert distortion_instance.maptype == "Dexela_2923"
+    assert distortion_instance.is_trivial
+
+    # Test invalid length raises
+    with pytest.raises(AssertionError):
+        distortion_instance.params = [0.0] * 5  # Invalid length
+
+    # Test setting correct length
+    distortion_instance.params = np.array([1.0] * 8)
+    assert np.array_equal(distortion_instance.params, np.array([1.0] * 8))
+    distortion_instance.params = np.array([0.0] * 8)  # Reset
+
+    xy_in = np.array([[1.0, 2.0], [3.0, 4.0]])
+
+    # Trivial apply / apply_inverse
+    xy_out = distortion_instance.apply(xy_in)
+    np.testing.assert_array_equal(xy_out, xy_in)
+
+    xy_out_inv = distortion_instance.apply_inverse(xy_in)
+    np.testing.assert_array_equal(xy_out_inv, xy_in)
+
+    # Non-trivial apply / apply_inverse
+    distortion_instance.params = np.array([1.0] * 8)
+    xy_out_nontrivial = distortion_instance.apply(xy_in)
+    assert not np.array_equal(xy_out_nontrivial, xy_in)
+
+    xy_out_inv_nontrivial = distortion_instance.apply_inverse(xy_in)
+    assert not np.array_equal(xy_out_inv_nontrivial, xy_in)
```

Comment:
I wonder for tests like this if we should also verify that the output is correct? This currently just verifies that the numbers changed.

Reply:
We'll do it another time.